core/src/Kokkos_Macros.hpp

```diff
@@ -549,6 +549,15 @@ static constexpr bool kokkos_omp_on_host() { return false; }
 #define KOKKOS_ENABLE_CUDA_LDG_INTRINSIC
 #endif
 
+#if defined(KOKKOS_ENABLE_IMPL_CUDA_EMULATE_UNIFIED_MEMORY)
+#define KOKKOS_ENABLE_IMPL_CUDA_UNIFIED_MEMORY
+#endif
+
```

Comment:
I would move the `static_assert` here.

Reply:
Which static_assert?  The one about the CUDA runtime version in CUDA allocate?

Reply:
Yes

Reply:
I disagree, I think we should strive to avoid introducing too many backend specific details in that file.